features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtils.kt

```diff
@@ -107,7 +107,7 @@ internal class DrawableUtils(
             ?: bitmapWrapper.createBitmap(displayMetrics, width, height, config)
 
     internal companion object {
-        private const val MAX_BITMAP_SIZE_IN_BYTES = 10240 // 10kb
+        private const val MAX_BITMAP_SIZE_IN_BYTES = 15000 // 15kb
```

Comment:
notice that I've modified this in this pr, since it's a small change and relates directly to imageviews